.github/workflows/lint.yml

```diff
@@ -4,28 +4,18 @@
 name: Lint
 
 on:
-#   push:
-    # branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
 
 jobs:
-  build:
-
+  lint:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [15.x]
-
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm install
-    - name: Lint the main source
-      run: npx eslint src/*.js src/lib/*.js
-    - name: Lint the language grammars
-      run: npx eslint --no-eslintrc -c .eslintrc.lang.js src/languages/**/*.js
+      - uses: actions/checkout@v2
```

Comment:
Was this just entirely broken before and not even working?  Trying to understand the indent changes.

Reply:
No, either indentation works. Reverting to align with the GitHub actions documentation